apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerTestUtilsV2.java

```diff
@@ -527,7 +532,7 @@ static void assertHcpcsCodes(
           "" + hcpcsYearCode.get(),
           null,
           hcpcsCode.get(),
-          item.getService().getCoding());
```

Comment:
While I appreciate the fixes in this class, they seem out of scope in an already large PR. (I'd recommend self-annotating your review when its large to help point out some of these things to help reviewers know what the deal is with seemingly-unrelated code changes)

Reply:
point taken. I was absolutely dismayed at the number of TODO's everywhere...probably hit the wall when I changed that one!

Reply:
reverted back to being commented out.